cmd/print.go

```diff
@@ -326,8 +368,11 @@ func newPrintCommand() *cobra.Command {
 	cmd.Flags().StringVarP(&pr.outputFormat, "output", "o", "yaml",
 		"Output format. One of: (yaml, json, kyaml).")
 
-	cmd.Flags().StringVar(&pr.inputFile, "input-file", "",
-		`Path to the manifest file. When set, the tool will read ingresses from the file instead of reading from the cluster. Supported files are yaml and json.`)
+	cmd.Flags().StringSliceVar(&pr.inputPaths, "input-file", []string{},
+		`Path to manifest file(s) or directory(s). When set, the tool will read ingresses from the specified files or all YAML files in directories. Supported files are yaml and json.`)
+
+	cmd.Flags().BoolVar(&pr.recursive, "recursive", false,
```

Comment:
This one is more personal, but I think we can omit this path since users can already do `-input-file manifest-dir/**/*.{yaml,yml,json}`, but if this is a common enough pattern in other places I'm happy to include.